packages/core-svelte/src/components/TextDropdownComponent.svelte

```diff
@@ -305,14 +305,11 @@
      * original value.
      */
     const endEditing = () => {
-        LOGGER.log('TextDropdownComponent: endEditing');
-        if (!isEditing) {
```

Comment:
There was a reason why we (together!) added this if-stat. Are you sure about this change?

Reply:
No, reverted the change.